tests/integration/adapter/store-adapter-test.js

```diff
@@ -24,6 +24,15 @@ var set = Ember.set;
 var run = Ember.run;
 var Person, Dog, env, store, adapter;
 
+function moveRecordOutOfInFlight(record) {
+  run(function() {
+    // move record out of the inflight state so the tests can clean up
+    // correctly
```

Comment:
It's because of [this assertion](https://github.com/emberjs/data/blob/41193f72435eca264ff3941134d2af1c72f6a861/addon/-private/system/model/states.js#L433-L437) in the state machine.